handlers/deploy.go

```diff
@@ -110,12 +128,42 @@ func deploy(ctx context.Context, req types.FunctionDeployment, client *container
 
 	log.Printf("Container ID: %s\tTask ID %s:\tTask PID: %d\t\n", container.ID(), task.ID(), task.Pid())
 
-	ip, netErr := getIP("netns0", int(task.Pid()))
+	id := uuid.New().String()
+	netns := fmt.Sprintf(NetNSPathFmt, task.Pid())
+
+	cni, err := gocni.New(gocni.WithPluginConfDir(CNIConfDir),
+		gocni.WithPluginDir([]string{CNIBinDir}))
+
+	if err != nil {
+		return err
+	}
+
+	// Load the cni configuration
+	if err := cni.Load(gocni.WithLoNetwork, gocni.WithDefaultConf); err != nil {
+		log.Fatalf("Failed to load cni configuration: %v", err)
+	}
+
+	labels := map[string]string{}
+
+	result, err := cni.Setup(ctx, id, netns, gocni.WithLabels(labels))
+	if err != nil {
+		log.Fatalf("failed to setup network for namespace %q: %v", id, err)
```

Comment:
I wonder if this should not just return an error, rather than kill the whole process?

Reply:
Done!